packages/fetchai/skills/aries_alice/skill.yaml

```
@@ -52,9 +52,15 @@ models:
     args:
       admin_host: 127.0.0.1
       admin_port: 8031
+      classification:
+        piece: classification
+        value: alice
```

```suggestion
        value: identity.aries.alice
```
seems more useful, same for faber